objective-c.html.markdown

```diff
@@ -128,9 +128,10 @@ int main (int argc, const char * argv[])
     // May contain different data types, but must be an Objective-C object
     NSArray *anArray      = @[@1, @2, @3, @4];
     NSNumber *thirdNumber = anArray[2];
-    NSLog(@"Third number = %@", thirdNumber); // Print "Third number = 3"
-    // NSMutableArray is mutable version of NSArray allowing to change items in array
-    // and extend or shrink array object. Convenient, but not as efficient as NSArray
+    NSLog(@"Third number = %@", thirdNumber); // prints "Third number = 3"
+    // NSMutableArray is a mutable version of NSArray allowing you to change 
```

Comment:
While we're at it:

NSMutableArray is a mutable version of NSArray, allowing you to change the items in the array and to extend or shrink the array object. Convenient, but not as efficient as NSArray.